slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/session/model/Session.java

```diff
@@ -0,0 +1,52 @@
+package com.binarystudio.academy.slidez.domain.session.model;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import com.binarystudio.academy.slidez.domain.presentation.model.Presentation;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "sessiones")
+public class Session {
+
+	@Id
+	@GeneratedValue(generator = "UUID")
+	@GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+	@Column(name = "id", updatable = false, nullable = false)
+	private UUID id;
+
+	@OneToOne(cascade = CascadeType.ALL)
```

Comment:
Do we have one session per one presentation? The session may change later for the same presentation. If not, let me know this)

Reply:
I think so. Probably in future we will change it.